docs/faq.md

```diff
@@ -63,4 +63,68 @@ The gene expression data files available for download report all possible genes
 ## Where can I see the code for generating QC reports? 
 
 A QC report for every processed library is included with all downloads, generated from the unfiltered and {ref}`filtered <processing_information:filtering cells>` {ref}`gene expression files <gene_expression_file_contents:gene expression file contents>`.
-You can find the [function for generating a QC report](https://github.com/AlexsLemonade/scpcaTools/blob/main/R/generate_qc_report.R) and the [QC report template documents](https://github.com/AlexsLemonade/scpcaTools/tree/main/inst/rmd) in the package we developed for working with processed ScPCA data, [`scpcaTools`](https://github.com/AlexsLemonade/scpcaTools). 
\ No newline at end of file
+You can find the [function for generating a QC report](https://github.com/AlexsLemonade/scpcaTools/blob/main/R/generate_qc_report.R) and the [QC report template documents](https://github.com/AlexsLemonade/scpcaTools/tree/main/inst/rmd) in the package we developed for working with processed ScPCA data, [`scpcaTools`](https://github.com/AlexsLemonade/scpcaTools). 
+
+## What if I want to use Seurat instead of Bioconductor? 
+
+The files available for download contain [`SingleCellExperiment` objects](http://bioconductor.org/books/3.13/OSCA.intro/the-singlecellexperiment-class.html). 
+If desired, these can be converted into Seurat objects. 
+
+You will need to [install and load the `Seurat` package](https://satijalab.org/seurat/articles/install.html) to work with Seurat objects.
+
+For libraries that only contain RNA-sequencing data (i.e. do not have a CITE-seq library found in the `altExp` of the `SingleCellExperiment` object), you can use the following commands:
+
+```
+library(Seurat)
+library(SingleCellExperiment)
+
+# read in RDS file 
+sce <- readRDS("SCPCL000000_filtered.rds")
+
+# grab counts matrix from SingleCellExperiment needed to build the Seurat Object
+counts <- counts(sce)
+
+# colData and rowData will need to be converted to a data.frame to be used in the Seurat object
+cell_metadata <- as.data.frame(colData(sce))
+row_metadata <- as.data.frame(rowData(sce))
+
+# create seurat object 
+seurat_object <- CreateSeuratObject(counts = counts, 
+                                    assay = "RNA",
+                                    project = "SCPCL000000")
```

Comment:
A few lines in here were duplicated later, where I think they make more sense (after explanation of why we need them). 

I also removed pulling a separate `counts` object, as that is the biggest part, so it would be better to encourage fewer copies of that matrix.

```suggestion
# create seurat object from the SCE counts matrix
seurat_object <- CreateSeuratObject(counts = counts(sce), 
                                    assay = "RNA",
                                    project = "SCPCL000000")
```